src/components/tasks/TaskDebuggingInformation.tsx

```diff
@@ -39,11 +44,27 @@ function TaskDebuggingInformation(props: Props) {
           let prettyTime = new Date(event.timestamp).toLocaleTimeString();
           return `${prettyTime} ${event.message}`;
         });
+
+        const agentNotificationsComponent =
+          !task.executionInfo.agentNotifications || task.executionInfo.agentNotifications.length === 0 ? null : (
+            <>
+              <Typography variant="subtitle1" sx={{ pt: 2 }}>
```

Comment:
Why not `ListSubheader` as I mentioned initially? Doesn't look good?

Reply:
It doesn't look consistent to the other headers provided by the `InlineLogs`, yes.

Reply:
What do you think of extracting styles into a top level variable so it's consistent with how we usually style components?